src/python/ensembl/io/genomio/manifest/compute_stats.py

```diff
@@ -101,8 +102,10 @@ def run(self, stats_path: PathLike):
             stats_out.write("\n".join(stats))
 
         # Die if there were errors in stats comparison
-        if self.error:
-            raise StatsError(f"Stats count errors, check the file {stats_path}")
+        if self.errors:
+            with open(self.errors_file, "w") as errors_fh:
```

Comment:
```suggestion
            with self.errors_file.open("w") as errors_fh:
```
Just to use the `Path()` features.